source/simulator/initial_conditions.cc

```diff
@@ -270,11 +270,21 @@ namespace aspect
           ComponentMask property_mask  (particle_property_manager->get_data_info().n_components(),false);
           property_mask.set(particle_property,true);
 
-          const std::vector<std::vector<double> > particle_properties =
-            particle_interpolator->properties_at_points(particle_postprocessor.get_particle_world().get_particle_handler(),
-                                                        quadrature_points,
-                                                        property_mask,
-                                                        cell);
+          std::vector<std::vector<double> > particle_properties;
+
+          try
+            {
+              particle_properties =
+                particle_interpolator->properties_at_points(particle_postprocessor.get_particle_world().get_particle_handler(),
+                                                            quadrature_points,
+                                                            property_mask,
+                                                            cell);
+            }
+          catch (const std::exception &e)
+            {
+              std::cerr << e.what() << std::endl;
+              throw e;
```

Comment:
This means that you will print the same error message twice. How about throwing `ExcQuiet` instead?

In some sense, what you're doing here is just adding debug output. In reality, if I understand correctly, you already found what the real problem is (namely calling `properties_at_point()` for invalid points). Do you even still need this patch then?